pagai/views.py

```diff
@@ -30,8 +24,12 @@ def explore(resource_id, table):
     query params (eg: /explore/<resource_id>/<table>?first=10).
     """
     limit = request.args.get("first", 10, type=int)
+    # Get headers
+    authorization_header = request.headers.get("Authorization")
+    id_token = request.headers.get("IdToken")
 
-    resource = get_pyrog_client().get_resource(resource_id)
+    pyrog_client = pyrog.PyrogClient(authorization_header, id_token)
```

Comment:
should we raise an error if we miss either auth_token or id_token ? we may have a hard time understanding the error otherwise

Reply:
Right! I think the id token is not mandatory because pyrog-server can introspect the auth token
I'll add something to raise an error here